### DISCUSSION TITLE
Unicode autocompletion does not work properly

### DISCUSSION THREAD

**Discussion by Rudolfo00** [zed-industries/zed#13901] [Feature Requests]
<img width="386" alt="image" src="https://github.com/zed-industries/zed/assets/171352525/e316af13-329d-4177-a7c0-813cdac5612d">

I want that `\alpha` is displayed at the top since this is also the most similar to the expression I typed. This is in the context of Julia.

**Comment by shirazkn** [comment]
Are you using the Unicode extension? The extension doesn't work for me. I type in "\alpha" but it won't open a drop-down list.

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
